docs/src/man/whatisasurrogate.md

```diff
@@ -34,4 +23,20 @@ s = surrogate(ts, IAAFT())
 surroplot(ts, s)
 ```
 
+## Performing surrogate hypothesis tests
+
+A surrogate test is a statistical test of whether a given timeseries satisfies or not a given hypothesis regarding its properties or origin.
+
+For example, the first surrogate methods were created to test the hypothesis,
+whether a given timeseries `x` that appears noisy may be the result of a linear
+stochastic process or not. If not, it may be a nonlinear process contaminated with observational noise. For the suitable hypothesis to test for, see the documentation strings of provided `Surrogate` methods or even better the review from Lancaster et al. (2018)[^Lancaster2018].
```

Comment:
```suggestion
stochastic process or not. If not, it may be a nonlinear process contaminated with observational noise. For the suitable hypothesis to test for, see the documentation strings of provided `Surrogate` methods or, even better, the review from Lancaster et al. (2018)[^Lancaster2018].
```